### PR TITLE
More enzyme test updates

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[More enzyme test updates](https://trello.com/c/bSojSOmJ/403-more-enzyme-test-updates)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8573/

File: jujugui/static/gui/src/app/components/copy-to-clipboard/test-copy-to-clipboard.js [hunk 1]
```diff
@@ -4,41 +4,56 @@
...
 describe('CopyToClipboard', function() {
+
+  const renderComponent = (options = {}) => {
+    return enzyme.shallow(
+      <CopyToClipboard
+        className={options.className}
+        value={options.value} />,
+      // Don't call componentDidMount as it requires nodes that don't exist in
+      // the shallow renderer.
+      { disableLifecycleMethods: true });
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/copy-to-clipboard/test-copy-to-clipboard.js]: cool

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**